CONTRIBUTING.md

```diff
@@ -10,10 +10,10 @@ cd titiler
 
 python -m pip install \
    pre-commit \
-   -e src/titiler/core["test"] \
-   -e src/titiler/extensions["test,cogeo,stac"] \
-   -e src/titiler/mosaic["test"] \
-   -e src/titiler/application["test"]
+   -e "src/titiler/core[test]" \
+   -e "src/titiler/extensions[test,cogeo,stac]" \
+   -e "src/titiler/mosaic[test]" \
+   -e "src/titiler/application[test]"
```

Comment:
I think this was just some invalid syntax. It tripped me up so I thought I'd sneak this change in to help others.

Reply:
> This is done automatically in CI https://github.com/developmentseed/titiler/blob/main/.github/workflows/deploy_mkdocs.yml#L16-L61
you just need to update https://github.com/developmentseed/titiler/blob/main/docs/src/advanced/dependencies.md and the endpoints options https://github.com/developmentseed/titiler/blob/main/docs/src/endpoints/cog.md

How about this? 941acd8